base/applications/cmdutils/more/lang/ro-RO.rc

```diff
@@ -1,60 +1,65 @@
 /*
- * FILE:       base/applications/cmdutils/more/lang/ro-RO.rc
- *             ReactOS Project (https://reactos.org)
- * TRANSLATOR: Fulea Ștefan (PM on ReactOS Forum at fulea.stefan)
- * CHANGE LOG: 2011-08-20  initial translation
- *             2011-10-17  diacritics change, other minor changes
+ * PROJECT:     ReactOS More Command
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2011-2019 Ștefan Fulea <stefan.fulea@mail.com>
+ *              Copyright 2023 Andrei Miloiu <miloiuandrei@gmail.com>
  */
 
 LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
 
 STRINGTABLE
 BEGIN
-    IDS_USAGE "Afișează date pe ecran, pagină-cu-pagină.\n\
+    IDS_USAGE "Afișarea datelor pe ecran, pagină cu pagină.\n\
 \n\
-MORE [/E [options]] < [drive:][path]filename\n\
-command | MORE [/E [options]]\n\
+MORE [/E [opțiuni]] < [disc:][cale]numele fișierului\n\
+command | MORE [/E [opțiuni]\n\
 \n\
-    [drive:][path]filename  A file whose contents are to be displayed.\n\
-    command                 A command whose output is to be displayed.\n\
+    [disc:][cale]numele fișierului  Un fișier al cărui conținut urmează\n\
+                                    să fie afișat.\n\
+    comandă                         O comandă a cărei ieșire urmează să fie afișată.\n\
```

Comment:
This line is over 80 characters long.

Reply:
We do not forbid that in rc files (yes, we also do not recommend it neither). But keeping the newlines similar to or same as en-US is more important than the lines lengths in rcs.

Reply:
So, should I revert this change?

Reply:
Not needed. I resolved the conversation, as you most likely noticed. I left that comment more as a general note.